app/services/qa/linked_data/ldpath_service.rb

```diff
@@ -0,0 +1,38 @@
+# Defines the external authority predicates used to extract additional context from the graph.
+require 'ldpath'
+
+module Qa
+  module LinkedData
+    class LdpathService
+      VALUE_ON_ERROR = [].freeze
+
+      # Create the ldpath program for a given ldpath.
+      # @param [String] ldpath to follow to get a value from a graph (documation: http://marmotta.apache.org/ldpath/language.html)
+      # @param [Hash] shortcut names for URI prefixes with key = part of predicate that is the same for all terms (e.g. { "madsrdf": "http://www.loc.gov/mads/rdf/v1#" })
```

Comment:
You are missing the term `prefixes` here after `[Hash]`

Reply:
Double checked [yardoc documentation](https://www.rubydoc.info/gems/yard/file/docs/Tags.md#param).    For named params, the param name goes before the type.  I'll update the yardoc in this file and others already touched by this PR.

I might need to check other places to see if it is used consistently.  But I'll do that under a separate PR.

Reply:
Turns out this was the only file effected.